the-content-guide/readme.md

```diff
@@ -7,14 +7,100 @@ The target audience are communication professionals. See it as a comprehensive g
+
+One goal of content marketing is that businesses want to increase traffic. Top named challenges and priorities for marketing professionals –besides creating more _high quality content_ in lesser time— are:
+
+* Measure content effectiveness
+* Producing more _engaging_ content
+* Metrics that measure Sales lead quality, sales and higher conversion rates.
+
+(sources survey's: [Content marketing institute](http://contentmarketinginstitute.com/wp-content/uploads/2015/09/2016_B2B_Report_Final.pdf), Linkedin B2B content marketing report, [Hubspot's The State of Inbound 2015](http://www.stateofinbound.com/))
+
+To meet and tackle these challenges and priorities, content has to be found. _Search Engine Optimisation_ or keyword research strategies help to create a more visible blogpost by readers and enabling businesses to get a lot more traffic.
+
+<img src="https://media0.giphy.com/media/KJg6Znn4V1Jcs/200.gif" width="100%">
+
+### Understanding your target audience for SEO
+
+As a content marketeer, writer or other creative, you sometimes wish you could see in the brains of your target audience. With almost everyone connected with the internet, people are using search engines as Google to look for answers or solely entertainment. Nothing new here, but it's important to recognise that current technology enables us to analyse the search behavior of potential audiences. We can therefor use this information for searchable content.
+
+So, implementing the right keywords will get you more traffic from your target audience on your blog. Apart from that, it's just as important and an advantage to really know what your audience wants.   
+
+## No image? Forget about it...
+
+Humans are visual creatures. Research done by [Skyword](http://www.skyword.com/contentstandard/enterprise-marketing/skyword-study-add-images-to-improve-content-performance/) has shown that chances a text will be read are increased with **94%** if the article shown has an image.
+Photo's and illustrations together with the right title make the reader curious and excited.
+
+> 94% is the growth that an article goes through if it has a relevant main image, compared to an article that has no image at all.
+
+Most important is that the images enriches the article in the sense that they serve the written piece. Apart from esthetics images should create a certain amount of context. So even though images are really important in any article, a image that makes no sense can be way worse than no image at all.
+
+A good mantra, with the addition of more multimedia in the back of our minds, is not "form follows content" but "content follows form".
+
+## Ending
+
+Our content developers have aquired all these qualities in their own practice. Besides that, each one of are team member contributes to every post made, which makes sure that all these diverse qualities are brought together in one big whole, which makes for the best content possible, which is pretty awesome.
+
```

Comment:
"Besides that" is not really proper English. Suggestions: "Additionally" or "Moreover"

"makes sure"--> "ensures" again.